ecl/hql/hqlexpr.cpp

```diff
@@ -8156,7 +8288,17 @@ IHqlExpression *CHqlRemoteScope::lookupSymbol(IIdAtom * searchName, unsigned loo
     if (resolvedSym && resolvedSym->getOperator() == no_processing)
     {
         if (lookupFlags & LSFrequired)
-            throwRecursiveError(searchName);
+        {
```

Comment:
I don't think this is likely to improve the error message - do you have examples where it does?